src/makeRobot.js

```diff
@@ -38,7 +38,61 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    info: `name: ${name}, chip version: ${version}, wheels: ${wheels}`,
+    coords:
+  {
+    x: 0, y: 0,
+  },
```

Comment:
Robot should have this 3 fielse insted of info field
```suggestion
  const robot = {
    name,
    version,
    wheels,
    coords: {
      x: 0,
      y: 0,
    },
```